app/assets/javascripts/controllers/container_topology/container_topology_controller.js

```diff
@@ -3,24 +3,40 @@
 miqHttpInject(angular.module('topologyApp', ['kubernetesUI', 'ui.bootstrap', 'ManageIQ']))
 .controller('containerTopologyController', ContainerTopologyCtrl);
 
-ContainerTopologyCtrl.$inject = ['$scope', '$http', '$interval', '$location', 'topologyService'];
+ContainerTopologyCtrl.$inject = ['$scope', '$http', '$interval', '$location', 'topologyService', '$window'];
+
+function ContainerTopologyCtrl($scope, $http, $interval, $location, topologyService, $window) {
+  $(function($) {
+    $(document).ready(function() {
+      miqHideSearchClearButton();
+    });
+  });
+
+  // Override breadcrumb height. Space is expensive in this page.
+  breadcrumb = document.getElementsByClassName("breadcrumb")[0]
+  if (breadcrumb != null) breadcrumb.style.marginBottom = '0px';
```

Comment:
> can this be done in css instead of in js?
> can it also be generalized to apply to middleware and network topology as well not to copy paste it later?

I really couldnt find a way to do this through css since the breadcumb isnt under the `container_topology` class. 
@himdel Any ideas?